### PR TITLE
Move post_update to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move include/post_update.php to src class

Related to #3878

### CODE REVIEW

**Comment by annando** [comment]
I was thinking about where to move it. @MrPetovan do you agree with this place? I was thinking to move it there as well, but was unsure.

File: src/Worker/CronJobs.php [hunk 1]
```diff
@@ -39,9 +36,9 @@ public static function execute($command = '')
 		logger("Starting cronjob " . $command, LOGGER_DEBUG);
 
 		// Call possible post update functions
-		// see include/post_update.php for more details
+		// see src/Database/PostUpdate.php for more details
```
Comment (MrPetovan) [inline src/Worker/CronJobs.php]: There isn't much more detail about how to use post updates!

It makes me think I should have used them for the password and plugin updates. 😅

Comment (annando) [reply src/Worker/CronJobs.php]: The post updates are for stuff than doesn't block the working with the system.

Comment (MrPetovan) [reply src/Worker/CronJobs.php]: Nevermind them, I'm specializing myself in blocking the system.

**Pull request merged**